entities.csv

```diff
@@ -264,6 +264,7 @@ Mitochondrial_Ribosome
 MRPS
 MRPL
 MMP
+MT

```

Comment:
It might make sense to use Metallothionein as the name

Reply:
OK. I went with the shortform because this is a very common grounding for MT, but the full name is more descriptive

Reply:
Yeah, MT might be okay too, @johnbachman what do you think?

Reply:
![image](https://user-images.githubusercontent.com/1953382/56438021-86b02380-62af-11e9-90c0-55ea4122e1bb.png)

Reply:
it's a close second to microtubule